From d5c8611e12156ca52e61117e6faa346e159a3618 Mon Sep 17 00:00:00 2001
From: StaticRocket <35777938+StaticRocket@users.noreply.github.com>
Date: Tue, 25 Apr 2023 22:51:14 -0500
Subject: [PATCH] cargo: patch the source to pull a patch

Update the source to pull a local copy of uhid-virt with a patch to
resolve "reference to packed field is unaligned" until flukes fork of
uhid-virt accepts it.

Signed-off-by: StaticRocket <35777938+StaticRocket@users.noreply.github.com>
---
 Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 6278384..ad1d1a3 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -39,6 +39,9 @@ versions = "4.1"
 
 notify-rust = { git = "https://github.com/flukejones/notify-rust.git", default-features = false, features = ["z"] }
 
+[patch.crates-io]
+uhid-virt = { path = "../uhid-virt" }
+
 [profile.release]
 # thin = 57s, asusd = 9.0M
 # fat = 72s, asusd = 6.4M
-- 
2.40.0

